src/decimal.rs

```diff
@@ -273,6 +276,73 @@ impl TryMul<Decimal> for Decimal {
     }
 }
 
+#[cfg(feature = "ops-traits")]
+impl<T> Mul<T> for Decimal
+where
+    T: Into<U192>,
+{
+    type Output = Decimal;
+    fn mul(self, rhs: T) -> Decimal {
+        self.try_mul(rhs).unwrap()
+    }
+}
+
+#[cfg(feature = "ops-traits")]
+impl Mul<Decimal> for Decimal {
```

Comment:
We could also enjoy the "assign" variants like `MulAssign` wdyt?

Reply:
the thing is that we don't use it anywhere inside our code, so not sure if it's really a need tbh